R/calculate.cis.matrix.R

```diff
@@ -80,9 +80,12 @@ calculate.cis.matrix <- function(
 		nrow = nrow(per.patient.data.type.corr[[1]]),
 		ncol = ncol(per.patient.data.type.corr[[1]])
 		);
-	for(i in 1:nrow(median.per.patient.data.type.corr)) {
-		for(j in 1:ncol(median.per.patient.data.type.corr)) {
-			median.per.patient.data.type.corr[i,j] <- median(sapply(per.patient.data.type.corr, function(x) {x[i,j]}));
+	for (i in 1:nrow(median.per.patient.data.type.corr)) {
+		for (j in 1:ncol(median.per.patient.data.type.corr)) {
+			median.per.patient.data.type.corr[i,j] <- median(sapply(per.patient.data.type.corr, function(x) {
```

Comment:
I added the `na.rm` here just in case.
The `apply.scaling` function doesn't allow any `NA`, so it would be okay if the user used `apply.scaling` before.